Quran/HighlightingView.swift

```diff
@@ -8,12 +8,22 @@
 
 import UIKit
 
+@objc protocol HighlightingViewDelegate {
```

Comment:
because it will be used as a `weak` reference to avoid retain cycle. You can't mark non ObjC protocols as weak as far as I know.